### PR TITLE
Fix summarization, rename to comprehensiveness

### PR DESCRIPTION
**Pull request by joshreini1**
Fixes https://github.com/truera/trulens/issues/811

Screenshot 2024-01-23 at 10 09 10 AM

### CODE REVIEW

File: trulens_eval/trulens_eval/feedback/prompts.py [hunk 1]
```diff
@@ -121,7 +121,7 @@
 
 STEREOTYPES_PROMPT = v2.Stereotypes.prompt.template
 
-SUMMARIZATION_PROMPT = """
```
Comment (daniel-huang-1230) [inline trulens_eval/trulens_eval/feedback/prompts.py:124]: just curious - what's the reason for renaming summarization prompt to comprehensiveness?

Comment (joshreini1) [reply trulens_eval/trulens_eval/feedback/prompts.py:124]: Talking to users and customers, comprehensiveness is the standard way that this eval is commonly described. It's also more descriptive than "summarization" on what it does.

File: trulens_eval/trulens_eval/feedback/prompts.py [hunk 2]
```diff
@@ -131,26 +131,22 @@
...
 Step 2 - Compare:
 (DO NOT SKIP THIS STEP!)
 
-For a SUMMARY How well does this summary address the above main points?
-
-please answer with this template:
-
-(Step 2)
-Supporting Evidence: <For each of the Important Points, explain if the SUMMARY does or does not mention it.>
-Score: <Give a score from 0 to 10 on if the SUMMARY addresses every single one of the main points. A score of 0 is no points were mentioned. A score of 5 is half the points were mentioned. a score of 10 is all points were mentioned.>
```
Comment (daniel-huang-1230) [inline trulens_eval/trulens_eval/feedback/prompts.py:146]: Do we not need this rating rubric any more?

Comment (joshreini1) [reply trulens_eval/trulens_eval/feedback/prompts.py:146]: Just using the standard COT template instead.

**Review by daniel-huang-1230 — APPROVED** [review]

**Pull request merged**